src/templates/wizardHeader/form.ejs

```diff
@@ -14,8 +14,23 @@
       <span data-icon="square" class="pr-1 fg-slate"></span>
       {% } %}
 
-      {% var color = active || completed ? "fg-slate" : ""  %}
-      {% var style = active ? "fw-medium" : completed ? "u" : "" %}
+      <!-- {% var color = active || completed ? "fg-slate" : ""  %} -->
+      <!-- {% var style = active ? "fw-medium" : completed ? "u" : "" %} -->
+      {% if (active || completed) {
+        color = "fg-slate";
```

Comment:
`color` and `style` are being declared as global variables since they aren't prefixed with `var`, `const`, or `let`. This can be addressed by adding `var` to the variable definition like this in each place they are set:

```suggestion
        var color = "fg-slate";
```

Another option would be to declare them before the `if` statements, which also means that the `else` clause can be left off [since Nunjucks doesn't render anything for a null value](https://mozilla.github.io/nunjucks/templating.html#variables):

> If a value is `undefined` or `null`, nothing is displayed. The same behavior occurs when referencing `undefined` or `null` objects. 

Here's what that would look like:

```javascript
{% var color;
  if (active || completed) {
    color = "fg-slate";
  }
%}
```

Reply:
Just implemented your last suggestion. I'm curious about the statement with 'else if'. Does the change I made to that one follow convention?

Reply:
@NamibiaLT It looks good!